### PR TITLE
Use `Authorization` header instead of `access_token` URL parameter

### PR DESCRIPTION
**Pull request by dan2k3k4**
Fix the issue from download-directory/download-directory.github.io#31 to switch from URL query parameters to Authorization HTTP header instead for the Github fetch call.

### CODE REVIEW

**Review by fregante — CHANGES_REQUESTED** [review]
If the token is missing GitHub will complain though.  The header can’t be there at all if there’s no token

**Comment by dan2k3k4** [comment]
@fregante updated, perhaps not the most elegant approach. I assume an empty header object can still be passed in?

File: index.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
 const fetch = require('node-fetch'); // Automatically excluded in browser bundles
 
 async function api(endpoint, token) {
-	token = token ? `&access_token=${token}` : '';
-	const response = await fetch(`https://api.github.com/repos/${endpoint}${token}`);
+	const headers = token ? {
+		Authorization: `Bearer ${token}`
+	} : {};
+	const response = await fetch(`https://api.github.com/repos/${endpoint}`, {
+		headers: headers
```
Comment (fregante) [inline index.js]: I think just this

```suggestion
		headers: token ? {Authorization: `Bearer ${token}`} : undefined
```

Comment (dan2k3k4) [reply index.js]: Updated 😄

**Comment by dan2k3k4** [comment]
@fregante any other changes that you'd like to see before this PR gets merged?

**Comment by fregante** [comment]
Sorry for the delay! Thanks for the PR. I'll try to release this as a breaking release, but I'd like to get #16 in too first.

**Pull request merged**